### ISSUE TITLE
Put "Why use SQL?" section on top

### ISSUE THREAD

**Issue by steltenpower** [data-lessons/library-sql#31]
Reader wants "is this for me?" answered asap or might leave before

**Comment by ndalyrose** [comment]
Agree. The way I would personally approach teaching 01-introduction to colleagues would be in this order: 

1. How/why do Librarians use SQL? Use Cases
2. Databases: From .CSV to Relational
3. What is SQL?
4. Database Management Systems 
5. Things to consider when choosing database management systems
6. Introduction to SQLite Manager

At British Library for instance we've got a ton of curators who are holding on to local databases either in .csv or software no longer supported (like ProCite) and they're always looking for information on how to migrate this content, so I'm keen to deliver this lesson onsite!
